src/layouts/UserLayout.less

```diff
@@ -1,15 +1,20 @@
 @import "~antd/lib/style/themes/default.less";
 
-.container {
-  background: #f0f2f5;
-  background-image: url('https://gw.alipayobjects.com/zos/rmsportal/TVYTbAXWheQpRcWDaDMu.svg');
-  width: 100%;
-  min-height: 100%;
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 100%;
-  padding: 110px 0 144px 0;
-  position: relative;
+.content {
+  padding: 32px 0;
```

Comment:
why 32px?

Reply:
8+8*3  according to the ant design guide 110 is to much in mobile, you want 24?

Reply:
actually 110 doesnt match antd 8+8*12 = 104,  8+8*13 = 112

Reply:
yep... 112 make sense

Reply:
PR is updated according to your comments!